content/using/running/vere.md

```diff
@@ -0,0 +1,632 @@
+
+### `-k, --key-file KEYS`
+
+Boot a new ship with the private key file `KEY`. See `-G` for an alternative.
+
+- Undocked: `urbit -w sampel-palnet -k /path/to/my.key`
+
+### `-K, --kernel-stage STAGE`
+
+Start at Hoon kernel version stage specified. This option is currently a no-op,
+and doesn't do anything. It may be revived in the future or it may be removed.
+
+### `-l, --lite-boot`
+
+Most-minimal startup. This option is currently a no-op, and doesn't do anything.
+It may be revived in the future or it may be removed.
+
+### `-L, --local`
+
+Run a ship with local networking only. This will disable Ames networking, so the
+ship will not be able to communicate with other ships. HTTP(S) will continue to
+work as normal.
+
+- Undocked: `urbit -L [pier]`
+- Docked: `[pier]/.run -L`
+
+### `-n, --replay-to NUMBER`
+
+Replay up to the specified event `NUMBER`.
+
+- Undocked: `urbit -n 10000 [pier]`
+- Docked: `[pier]/.run -n 10000`
+
+A snapshot will be saved in a jamfile at `[pier]/.urb/roc/[NUMBER].jam`.
+
+### `--no-conn`
+
+Disable the control pane unix socket.
+
+- Undocked: `urbit --no-conn [pier]`
+- Docked: `[pier]/.run --no-conn`
+
+### `--no-dock`
+
+From v1.8 onwards, when a new ship is booted, the urbit binary will copy itself
+into the newly created pier by default (see the [`dock`](#dock) section for
+details). To prevent this from happening, the `--no-dock` option can be used.
+
+- Undocked: `urbit --no-dock -w sampel-palnet -k /path/to/my.key`
+
+### `-p, --ames-port PORT`
+
+Set the Ames UDP port to `PORT`. This is the port used for inter-ship
+communications.
+
+- Undocked: `urbit -p 33333 [pier]`
+- Docked: `[pier]/.run -p 33333`
+
+### `-P, --profile`
+
+Enable profiling. Profiles will be saved to
+`[pier]/.urb/put/profile/[now].txt`. This is useful for debugging and
+development. Profiling hit counters in Hoon code specified with the `~$` rune
+will be recorded here.
+
+- Undocked `urbit -P [pier]`
+- Docked `[pier]/.run -P`
+
+### `-q, --quiet`
+
+Run a ship without verbosity. This disables the output that the `|verb` command
+toggles. This is the opposite of `-v`. Verbosity can be enabled again by running
+`|verb`.
+
+- Undocked: `urbit -q [pier]`
+- Docked: `[pier]/.run -q`
+
+### `-r, --replay-from NUMBER`
+
+Load snapshot at event `NUMBER`, only replay after that event. This lets you
+resume a partially completed replay. For the given `NUMBER`, there must be
+a corresponding snapshot in `[pier]/.urb/roc/[NUMBER].jam`, created by
+`cram` or `-n`.
+
+- Undocked: `urbit -r 10000 [pier]`
+- Docked: `[pier]/.run -r 10000`
+
+### `-R, --versions`
+
+Report Vere build info.
+
+```
+> urbit -R
+urbit 1.9
+gmp: 6.2.1
+sigsegv: 2.14
+openssl: OpenSSL 1.1.1n  15 Mar 2022
+libuv: 1.43.0
+libh2o: 0.13.6
+lmdb: 0.9.29
+curl: 7.81.0
+```
+
+- Undocked: `urbit -R [pier]`
+- Docked: `[pier]/.run -R`
+
+### `-S, --skip-battery-hashes`
+
+Disable battery hashing. This option is currently a no-op, and doesn't do
```

Comment:
¯\\_(ツ)_/¯ @joemfb do you know?

Reply:
(Answering the question here, not suggesting copy)

This argument disables hashboard, an alternate jet dashboard that validates jet registrations by the cryptographic hash of the battery. This option is always on (hashboard is always off), so it has no effect.

Hashboard itself works fine (although it can be slow in certain circumstances), but it implies a battery hash distribution channel that has never existed. It will most likely be replaced by other, newer jet dashboard designs.